README.md

```diff
@@ -13,20 +17,21 @@ gem 'solidus_feeds'
 Bundle your dependencies and run the installation generator:
 
 ```shell
-bundle
-bundle exec rails g solidus_feeds:install
+bin/rails generate solidus_feeds:install
 ```
 
 ## Usage
 
+<!-- Explain how to use your extension once it's been installed. -->
```

Comment:
> Did it slip in or is it here as a reminder to expand the description?

Same question as line 6 🤔

Reply:
That's part of the template offered by dev-support, leaving it there, given it's not visible in the readme, will make it easier to deal with diffs at the next update